scripts/variantstore/gvs-overview.md

```diff
@@ -0,0 +1,165 @@
+
+To import sample genomes, the GVS workflow calls two subworkflows detailed below.
+
+#### A. GvsAssignIds
+
+The [GvsAssignIds subworkflow (alias = AssignIds)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsAssignIds.wdl) assigns GVS-specific IDs to each of the samples in the sample set which are used as an access pattern. The IDs enable easy access to specific sample data. GvsAssignIds imports an additional workflow, [GvsCreateTables](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsCreateTables.wdl).
+
+#### B. GvsImportGenomes
+
+The [GvsImportGenomes subworkflow (alias = ImportGenomes)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsImportGenomes.wdl) validates that sample gVCF files contain required annotations and loads samples into several tables in BigQuery. GvsImportGenomes imports an additional workflow, [GvsUtils (alias = Utils)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsUtils.wdl).
+
+### 2. Train the filtering model
+
+To train the filtering model, the GVS workflow calls two subworkflows detailed below.
+
+#### A. GvsCreateAltAllele
+
+The [GvsCreateAltAllele subworkflow (alias = CreateAltAllele)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsCreateAltAllele.wdl) splits alternate alleles and calculates additional annotations to be used for filtering. GvsCreateAltAllele imports an additional workflow, [GvsUtils (alias = Utils)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsUtils.wdl).
+
+#### B. GvsCreateFilterSet
+
+The [GvsCreateFilterSet subworkflow (alias = CreateFilterSet)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsCreateFilterSet.wdl) creates a non-deterministic filtering model using [VQSR](https://gatk.broadinstitute.org/hc/en-us/articles/360035531612) and annotations from a random subset of the input samples. SNPs are recalibrated using the annotations `AS_FS`, `AS_ReadPosRankSum`, `AS_MQRankSum`, `AS_QD`, and `AS_SOR`. Indels are recalibrated using the annotations `AS_FS`, `AS_ReadPosRankSum`, `AS_MQRankSum`, `AS_QD`, `AS_SOR`, and `AS_MQ`. GvsCreateFilterSet imports two additional WDL scripts, [GvsWarpTasks (alias = Tasks)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsWarpTasks.wdl) and [GvsUtils (alias = Utils)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsUtils.wdl).
+
+### 3. Extract VCF files
+
+To extract joint VCF files, the GVS workflow calls two subworkflows detailed below.
+
+#### A. GvsPrepareRangesCallset
+
+The [GvsPrepareRangesCallset subworkflow (alias = PrepareRangesCallset)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsPrepareRangesCallset.wdl) transposes data into a new access pattern based on genomic coordinates.
```

Comment:
> transposes data into a new access pattern based on genomic coordinates.

I'm not sure what this would mean to beta users.  Maybe just better to say what it does at a high level like "subsets data into temporary tables for efficient extract" or something